image/db/rhel/konflux.Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM registry.redhat.io/rhel8/postgresql-12:latest
+
+LABEL \
+    com.redhat.component="rhacs-scanner-db-container" \
+    com.redhat.license_terms="https://www.redhat.com/agreements" \
+    description="Scanner Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.description="Scanner Database Image for Red Hat Advanced Cluster Security for Kubernetes" \
+    io.k8s.display-name="scanner-db" \
+    io.openshift.tags="rhacs,scanner-db,stackrox" \
+    maintainer="Red Hat, Inc." \
+    name="rhacs-scanner-db-rhel8" \
+    source-location="https://github.com/stackrox/scanner" \
+    summary="Scanner DB for RHACS" \
+    url="https://catalog.redhat.com/software/container-stacks/detail/60eefc88ee05ae7c5b8f041c" \
+    # We must set version label to prevent inheriting value set in the base stage.
+    # TODO(ROX-20236): configure injection of dynamic version value when it becomes possible.
+    version="0.0.1-todo"
+
+USER root
+
+COPY image/db/pg_hba.conf image/db/postgresql.conf /etc/
+COPY --chown=postgres:postgres image/db/rhel/scripts/docker-entrypoint.sh /usr/local/bin/
+
+RUN dnf upgrade -y --nobest && \
```

Comment:
this `dnf ugprade` isn't present downstream. I wonder if we can safely remove it.

Reply:
Well, it's there. See <https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-scanner-db/Dockerfile.in?ref_type=heads#L28>. We do this in all containers in order not to have vulnerabilities in containers: there's sometimes few weeks delay between RPMs are updated and corresponding base containers are released. By updating RPMs, we decrease the chance to go out with CVEs.